activemodel/lib/active_model/model.rb

```diff
@@ -58,42 +58,6 @@ module ActiveModel
   # (see below).
```

Comment:
Should we update the documentation to reflect what this module is supposed to be?

Reply:
I've updated the documentation and added a new section to the guides.